### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by ghost** [tnmendes/watt#50] [General]
First, I have never reached out to comment on an app before.  Probably never found an app that affected me this much so quickly.  After today, I can't say that anymore.  I even signed up for a Git account just so I could comment here.  Hopefully I'm not breaking any rules!
Second, I apologize for paying the minimum amount for the subscription.  That will change during the next renewal (which there definitely will be).
Found the app by chance yesterday in the App Store and within 5 minutes had all my devices talking to the app (not many devices yet).  Super easy to figure out the interface.  I then started thinking that it would be great if it could, [fill in feature here].
Found your website link in the App Store so I could request some features but found the Roadmap.  Every feature I wanted and more is already there!
You and/or your team deserve a "way to go" and "keep up the very good work"!  
Thanks for thinking so logically.  That's rare in apps like this.

Cheers!
Greg

**Comment by tnmendes** [comment]
Hey Greg, thank you for your kind words. I am really glad to know that the app is good hands with the users of watt and is useful in everyday life :D
